gitpod-io/website
# Optimize avatars images

## Description
Per discussion in https://github.com/gitpod-io/website/pull/2010 

Optimized avatars to be 96x96

## How to test
1. Go to homepage
2. Scroll down to testimonials and see if avatars are still there


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2013"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
